dashboard/config/locales/scripts.en.yml

```diff
@@ -25,6 +25,7 @@ en:
         cspexams_category_name: CS Principles Practice Test
         csp17_category_name: CS Principles ('17-'18)
         aiml_2021_category_name: AI / ML
+        csc_2021_category_name: CS Connnections ('21-'22)
```

Comment:
you'll probably have to undo this change to prevent a merge conflict with https://github.com/code-dot-org/code-dot-org/pull/43580

Reply:
i think i was wrong about this -- that PR has been merged and there don't appear to be any conflicts. i suppose because it's the exact same change, github worked it out automatically